sts/security-sts-server/src/main/java/ddf/security/sts/PropertyPlaceholderWrapper.java

```diff
@@ -45,10 +43,15 @@ public PropertyPlaceholderWrapper(DefaultConditionsProvider conditionsProvider,
             StaticSTSProperties properties) {
         samlConditionsProvider = conditionsProvider;
         stsProperties = properties;
+        // set the default values in case there is no configuration
+        samlConditionsProvider.setLifetime(1800);
+        stsProperties.setSignatureUsername("localhost");
+        stsProperties.setIssuer("localhost");
+        stsProperties.setEncryptionUsername("localhost");
```

Comment:
Just an FYI -- I am pretty sure the "localhost" and 1800 values will be marked as code violations (duplicate strings and magic numbers) in tools like SONAR although they aren't necessary issues. Not a big deal, but just something to keep in mind.